docs/node.md

```diff
@@ -80,7 +80,8 @@ npx babel-node --debug --presets es2015 -- script.js --debug
 | ---------------------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
 | `-e, --eval [script]`  |                             | Evaluate script                                                                                                                             |
 | `-p, --print`          |                             | Evaluate script and print result                                                                                                            |
-| `-i, --ignore [regex]` | `node_modules`              | Ignore all files that match this regex when using the require hook                                                                          |
+| `-i, --ignore [regex]` | `node_modules`              | Ignore all files that match this regex when using the require hook. If both -i and -o are omitted, any file outside current working directory is ignored too. |
+| `-o, --only [regex]`   | current working directory   | Include all files that match this regex when using the require hook. If both -i and -o are omitted, any file outside current working directory is ignored too. |
```

Comment:
Nit: `Only include the files` would be more explicit, since `Include all files` doesn't imply that what is not matched is excluded. Apart from that, the desc looks good :+1:

Reply:
I see. How about `Include only the files that match this regex when using the require hook and exclude the other files. If both -i and -o are omitted, any file outside current working directory is ignored too.` ?

(I'm not a native English speaker, so if you have a good alternative, please share it. Sorry in advance.)

Reply:
I'm not a native speaker either :stuck_out_tongue: 

Your proposed description seems ok to me.